### PR TITLE
#161967119  Add PostgreSQL as the DB engine for Authors Haven

### PR DESCRIPTION
**Pull request by Bryan-Cee**
#### What does this PR do?
Adds PostgreSQL as the DB engine for Authors Haven

#### Description of Task to be completed?
- Change the database settings in the settings file.
- Add psycopg2_binary to the requirements file

#### How should this be manually tested?

- Clone the repo and cd into the folder
- Create a virtual environment by running `$ python3 -m venv name_of_environment`
- Activate the virtual environment by running `$ source name_of_environment/bin/activate`
- Run `$ pip install -r requirements.txt`
- Then add the following environment variables to your computer.

  - `$ export DB_NAME='choose_a_dbname'`
  - `$ export DB_USER='db_username'`
  - `$ export DB_PASSWORD='db_role_password'`
  - `$ export DB_HOST='localhost or preffered host'`
  - `$ export DB_PORT='server_port_number'`

#### What are the relevant pivotal tracker stories?
[#161967119](https://www.pivotaltracker.com/story/show/161967119)

### CODE REVIEW

**Comment by Darius-Ndubi** [comment]
Change base branch to develop

File: authors/settings.py [hunk 1]
```diff
@@ -138,6 +142,6 @@
     'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
```
Comment (jillo-abdullahi) [inline authors/settings.py]: We're not dealing with any form of authentication at the moment. Perhaps we can leave out this part for now.

**Comment by jillo-abdullahi** [comment]
Otherwise, the PR looks okay to me.

**Comment by bettblake08** [comment]
File changes are up to standard. In the PR, I suggest you add the virtualenv activation as a step in the how-to-test section.

File: authors/settings.py (hunk 1 above)
Comment (Bryan-Cee) [reply authors/settings.py]: Lemme scrap it out

**Review by denomash — COMMENTED** [review]
Well done @Bryan-Cee. On settings.py from line 81 add a comment to make it clear with regard to the DATABASES

**Review by bettblake08 — COMMENTED** [review]
The PR is all good.

**Comment by davishooly** [comment]
@Bryan-Cee your badge for Travis in ReadMe file is pointing to master while the ReadMe for develop is pointing to develop that's why your badge is saying your build is not passing you can make changes to correct that

**Review by mutoromike — APPROVED** [review]
LGTM

**Pull request merged**